CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Changes
 
+## edge-20.7.1
+
+This edge release improves messaging for retries when pods are unschedulable,
+adds a feature to persist prometheus data as an option to storing the data in
```

Comment:
"as an option" reads a bit strangely to me.  did you mean "as an alternative"?

I'd word this more along the lines of: "added the option to use a persistent volume for Prometheus data, allowing data to persist across restarts"